crates/autopilot/src/boundary/mod.rs

```diff
@@ -39,4 +39,5 @@ pub struct SolvableOrders {
     pub orders: HashMap<domain::OrderUid, model::order::Order>,
     pub quotes: HashMap<domain::OrderUid, domain::Quote>,
     pub latest_settlement_block: u64,
+    pub fetched_from_db: chrono::DateTime<chrono::Utc>,
```

Comment:
why not to store it in timestamp?

Reply:
Do you mean why not use `std::time::Instant`? It's used in a DB query and that needs an actual timestamp string.